piksi_tools/console/settings.yaml

```diff
@@ -882,6 +902,26 @@
   Description: Number of ticks between GPGSA NMEA messages being sent
   Notes: This setting represents the integer number of solution periods between each transmission of the NMEA message. For example, if the solution rate is 10 Hz, and this rate setting is 2, the NMEA message will be sent every two solution epochs at a rate of 5 Hz.  If this setting is 0, the msg will be suppressed.
 
+- group: ntrip
+  name: enable
+  type: boolean
+  expert: true
+  units: N/A
+  default value: False
+  enumerated possible values: True,False
+  Description: Enable NTRIP client
+  Notes: If True, NTRIP client will be used.
+
+- group: ntrip
+  name: url
+  type: string
+  expert: true
+  units: N/A
+  default value: N/A
+  enumerated possible values:
+  Description: NTRIP URL to use
+  Notes: URL to use with NTRIP client. NTRIP must be enabled to use this setting.
```

Comment:
Can we explain what the format should be for the url?  at the least I would like to indicate http:// rather than ntrip://

Something like: The URL should take the form "http://XXX.com:PORT_NUMBER/mount_point". For example, http://swift.ntrip.com:2101/sf_office could be a valid ntrip URL